### ISSUE TITLE
Specifying the requirements addressed in Milestone 1

### ISSUE THREAD

**Issue by iremmer** [bounswe/bounswe2022group6#251]
We divided work into smaller pieces to complete the Milestone 1 Group report. 
I will check the requirements addressed in this Milestone and add them to our group report. 
To go with double check, @artun-akdogan will review after I finish.

Deadline 04.11.2022 22.00

**Comment by iremmer** [comment]
@artun-akdogan can you please review and make corrections if necessary?
You can reach it [here](https://github.com/bounswe/bounswe2022group6/blob/master/deliverables/CMPE451_Customer_Presentation_Milestone_1.md)

**Comment by artun-akdogan** [comment]
Thank you for your hard work. I have fixed the indentation of the 5th part. Also I have removed those parts from the report:

- 1.1.1.2.1 Registered users shall be able to add a phone number to their account.
- 1.1.1.3.2 Registered users shall be able to change the phone number in their account.
- 1.1.1.3.3 Registered users shall be able to change the profile picture in their account.

We didn't implemented features for phone numbers yet, and as far as I remember, we didn't implemented required features to change the profile picture of the account.

I have committed those changes, If any of those changes were inappropriate, I can revert them.
